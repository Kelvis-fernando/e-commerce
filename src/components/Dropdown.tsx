import { ReactNode } from "react";
import { DropdownContainer } from "../styles/dropdown";

interface DropdownProps {
  children: ReactNode;
  isOpen: boolean;
}

const Dropdown = ({ children, isOpen }: DropdownProps) => {
  return <DropdownContainer isOpen={isOpen}>{children}</DropdownContainer>;
};

export default Dropdown;
