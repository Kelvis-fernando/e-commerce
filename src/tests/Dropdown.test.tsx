import React from "react";
import { render } from "@testing-library/react";
import Dropdown from "../components/Dropdown";

describe("Dropdown", () => {
  test("should render and match snapshot", () => {
    const { container } = render(
      <Dropdown children={undefined} isOpen={false} />
    );

    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
