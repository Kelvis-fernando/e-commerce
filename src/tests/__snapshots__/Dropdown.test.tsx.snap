// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should render and match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe hnlUoq"
  />
</div>
`;
